narwhals/_pandas_like/dataframe.py

```diff
@@ -105,19 +105,30 @@ def __array__(self, dtype: Any = None, copy: bool | None = None) -> np.ndarray:
         return self.to_numpy(dtype=dtype, copy=copy)
 
     @overload
-    def __getitem__(self, item: tuple[Sequence[int], str | int]) -> PandasLikeSeries: ...  # type: ignore[overload-overlap]
+    def __getitem__(  # type: ignore[overload-overlap]
+        self, item: tuple[Sequence[int], Sequence[str], str | int]
+    ) -> PandasLikeSeries: ...
```

Comment:
I think we didn't need to change this one
```suggestion
    def __getitem__(self, item: tuple[Sequence[int], str | int]) -> PandasLikeSeries: ...  # type: ignore[overload-overlap]
```